packages.json

```diff
@@ -1231,7 +1246,34 @@
   "https://github.com/ibm-bluemix-mobile-services/bms-pushnotifications-serversdk-swift.git",
   "https://github.com/ibm-cloud-security/appid-serversdk-swift.git",
   "https://github.com/ibm-cloud-security/swift-jwk-to-pem.git",
+  "https://github.com/ibm-cloud-security/Swift-JWT-to-PEM.git",
   "https://github.com/IBM-Swift/alert-notification-sdk.git",
+  "https://github.com/IBM-Swift/BlueCryptor.git",
+  "https://github.com/IBM-Swift/BlueRSA.git",
+  "https://github.com/IBM-Swift/BlueSignals.git",
+  "https://github.com/IBM-Swift/BlueSocket.git",
+  "https://github.com/IBM-Swift/BlueSSLService.git",
+  "https://github.com/IBM-Swift/CCurl.git",
+  "https://github.com/IBM-Swift/CircuitBreaker.git",
+  "https://github.com/IBM-Swift/CloudEnvironment.git",
+  "https://github.com/IBM-Swift/Configuration.git",
+  "https://github.com/IBM-Swift/CZlib.git",
+  "https://github.com/IBM-Swift/HeliumLogger.git",
+  "https://github.com/IBM-Swift/Kitura-CouchDB.git",
+  "https://github.com/IBM-Swift/Kitura-Credentials.git",
+  "https://github.com/IBM-Swift/Kitura-net.git",
+  "https://github.com/IBM-Swift/Kitura-Session.git",
+  "https://github.com/IBM-Swift/Kitura-StencilTemplateEngine.git",
+  "https://github.com/IBM-Swift/Kitura-WebSocket.git",
+  "https://github.com/IBM-Swift/Kitura.git",
+  "https://github.com/IBM-Swift/LoggerAPI.git",
+  "https://github.com/IBM-Swift/OpenSSL.git",
+  "https://github.com/IBM-Swift/Swift-cfenv.git",
+  "https://github.com/IBM-Swift/swift-html-entities.git",
+  "https://github.com/IBM-Swift/Swift-JWT.git",
+  "https://github.com/IBM-Swift/Swift-Kuery.git",
+  "https://github.com/IBM-Swift/Swift-SMTP.git",
+  "https://github.com/IBM-Swift/SwiftyRequest.git",
```

Comment:
These are redirects - they shouldn't be appearing. I.e. it's

- https://github.com/Kitura/BlueCryptor

etc.

which are already in the list.

Reply:
Fixed in https://github.com/SwiftPackageIndex/Validator/commit/e279e972a8158e7c42d585e2c49176f2a959a4d5